src/main/scala/com/raquo/airstream/core/Observable.scala

```diff
@@ -100,7 +99,7 @@ trait Observable[+A] {
   def debugLogJs(prefix: String = "event", when: A => Boolean = _ => true): Self[A] = {
     map(value => {
       if (when(value)) {
-        dom.console.log(prefix + ": ", value.asInstanceOf[js.Any])
+        dom.console.log(prefix + ": " + value.asInstanceOf[js.Any])
```

Comment:
Unlike println, isn't an equivalent change. JS `console.log` prints out several parameters differently in a way that you can't achieve with string concatenation. For example, the old code would print an HTML element in a custom way that lets the user interact with the element, whereas the new code will just .toString the element.

Reply:
ok